packages/web-components/fast-foundation/docs/api-report.md

```diff
@@ -1345,42 +1344,61 @@ export class Listbox extends FoundationElement {
     protected _options: ListboxOption[];
     role: string;
     selectedIndex: number;
-    // (undocumented)
+    // @internal
     selectedIndexChanged(prev: number, next: number): void;
     selectedOptions: ListboxOption[];
-    // (undocumented)
-    protected selectedOptionsChanged(prev: any, next: any): void;
+    // @internal
+    protected selectedOptionsChanged(prev: ListboxOption[] | undefined, next: ListboxOption[]): void;
     selectFirstOption(): void;
     // @internal
     selectLastOption(): void;
     // @internal
     selectNextOption(): void;
     // @internal
     selectPreviousOption(): void;
-    // @internal (undocumented)
+    // @internal
     protected setDefaultSelectedOption(): void;
     protected setSelectedOptions(): void;
+    // @internal
+    protected shouldSkipFocus: boolean;
     static slottedOptionFilter: (n: HTMLElement) => boolean;
-    // @internal (undocumented)
-    slottedOptions: HTMLElement[];
-    // (undocumented)
-    slottedOptionsChanged(prev: any, next: any): void;
+    // @internal
+    slottedOptions: Element[];
+    // @internal
+    slottedOptionsChanged(prev: Element[] | unknown, next: Element[]): void;
     // @internal
     protected static readonly TYPE_AHEAD_TIMEOUT_MS = 1000;
-    // @internal (undocumented)
+    // @internal
     protected typeaheadBuffer: string;
-    // (undocumented)
+    // @internal
     typeaheadBufferChanged(prev: string, next: string): void;
+    // @internal @deprecated
+    protected get typeAheadExpired(): boolean;
+    protected set typeAheadExpired(value: boolean);
+    // @internal
+    protected typeaheadExpired: boolean;
     // @internal
-    protected typeAheadExpired: boolean;
-    // @internal (undocumented)
     protected typeaheadTimeout: number;
 }
 
 // @internal (undocumented)
 export interface Listbox extends DelegatesARIAListbox {
 }
 
+// @public
+export class ListboxElement extends Listbox {
+    // @internal @override
+    mousedownHandler(e: MouseEvent): boolean | void;
+    size: number;
```

Comment:
`size` and `sizeAttribute` need internal or public tags.

Reply:
They both do - the API extractor only marks non-public properties when they're `@internal`.